public/index.html

```diff
@@ -16,11 +18,145 @@
+                        <datalist id="category">
+                            <option>Bussiness</option>
+                            <option>Development</option>
+                            <option>Marketing</option>
+                        </datalist>
+                    </div>
+                    <div class="form-group col-md-4">
+                        <label for="text">Priority</label>
+                        <input type="text" class="form-control" id="#" list="Priority">
+                        <datalist id="Priority">
+                            <option>High</option>
+                            <option>Medium</option>
+                            <option>Low</option>
+                        </datalist>
+                    </div>
+                    <div class="form-group col-md-4">
+                        <label for="text">Difficulty</label>
+                        <input type="text" class="form-control" id="#" list="Difficulty">
+                        <datalist id="Difficulty">
+                            <option>High</option>
+                            <option>Medium</option>
+                            <option>Low</option>
+                        </datalist>
+                    </div>
+                </div>
+                <div class="form-row">
+                    <div class="form-group col-md-4">
+                        <label for="inputState">Status</label>
+                        <input type="text" class="form-control" id="#" list="statusCheck">
+                        <datalist id="statusCheck">
+                            <option>Ice Box</option>
+                            <option>Ready to start</option>
+                            <option>Under Progress</option>
+                            <option>Blocked</option>
+                            <option>Completed</option>
+                        </datalist>
+                    </div>
+                </div>
+
+                <button type="submit" class="btn btn-primary mt-2 mb-5">Submit</button>
+            </form>
+        </div>
+    </div>
+
+    <!-- JS, Popper.js, and jQuery -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+        integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+        integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+        crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
+        integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+        crossorigin="anonymous"></script>
+    <script src="script/workTrolly.js"></script>
```

Comment:
File workTrolly.js can be renamed to more specific name like workTrollyUI.js Since it is controlling the ui.

Reply:
okay, sure